.github/workflows/ci_workflows.yml

```diff
@@ -0,0 +1,92 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # Weekly Wednesday 7 AM build
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 7 * * 3'
+
+env:
+  TEST_BIGDATA: "https://bytesalad.stsci.edu/artifactory"
```

Comment:
Since the workflow cannot access our Artifactory server, probably no reason to have this defined.

Reply:
I think I had to define it because somewhere deep in `ci-watson`, it crashes if it is not there.

Reply:
But I don't quite remember now, so lemme remove it again and see.